MAUI/MauiAppBasic/MSALClient/PCAWrapper.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+
+namespace MauiAppBasic.MSALClient
+{
+    /// <summary>
+    /// This is a wrapper for PublicClientApplication. It is singleton and can be utilized by both application and the MAM callback
+    /// </summary>
+    public class PCAWrapper

```

Comment:
I would rename the class and file to `PublicClientApplicationWrapper`. It's longer but I feel like, especially for people new to MSAL, seeing the full name is better than the acronym. Or maybe name it `MsalWrapper`?

Reply:
That name is too long. The summary of the class spells PublicClientApplication. That should be sufficient.